index.js

```diff
@@ -184,6 +184,42 @@ var Space = redefine.Class({
     }
   },
 
+  // Space functions
+  /**
+   * Gets all locales enabled in space.
+   * @returns JSON of locale data for space
+   */
+  getLocales: function() {
+    return this.client.request('/spaces/' + this.sys.id + '/locales');
+  },
+
+
+  /**
+   * Creates a locale
+   * @param name  The readable name of the locale, e.g. U.S English
+   * @param code  the i18n abbreviation, e.g. en-US
+   */
+  createLocale: function(name, code) {
```

Comment:
I think that makes sense.  Would also remove the first line. However what
Should be the behavior if they submit an object with extra key:value
pairs?  Should we strip those?


On Wed, Oct 7, 2015 at 1:45 PM, Johannes Würbach <notifications@github.com>
wrote:

> In index.js
> <https://github.com/contentful/contentful-management.js/pull/33#discussion_r41421726>
> :
>
> > +  // Space functions
> > +  /**
> > +   * Gets all locales enabled in space.
> > +   * @returns JSON of locale data for space
> > +   */
> > +  getLocales: function() {
> > +    return this.client.request('/spaces/' + this.sys.id + '/locales');
> > +  },
> > +
> > +
> > +  /**
> > +   * Creates a locale
> > +   * @param name  The readable name of the locale, e.g. U.S English
> > +   * @param code  the i18n abbreviation, e.g. en-US
> > +   */
> > +  createLocale: function(name, code) {
>
> Shouldn't this method take an object as other create methods do?
> https://github.com/contentful/contentful-management.js/blob/master/index.js#L227
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/contentful/contentful-management.js/pull/33/files#r41421726>
> .
>